contracts/transmuter/libraries/LibGetters.sol

```diff
@@ -83,8 +83,11 @@ library LibGetters {
         // The `stablecoinsIssued` value need to be rounded up because it is then used as a divizer when computing
         // the amount of stablecoins issued
         stablecoinsIssued = uint256(ts.normalizedStables).mulDiv(ts.normalizer, BASE_27, Math.Rounding.Up);
-        if (stablecoinsIssued > 0)
-            collatRatio = uint64(totalCollateralization.mulDiv(BASE_9, stablecoinsIssued, Math.Rounding.Up));
-        else collatRatio = type(uint64).max;
+        if (stablecoinsIssued > 0) {
+            uint256 uncastCollatRatio = uint64(
```

Comment:
here it looks like we're still doing the casting to uint64

Reply:
Why not using safeCast otherwise?

Reply:
Because I don't want it to revert if it goes over this, but it may be considered risky actually